File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/legacy-extensions-codestarts/resteasy-codestart/scala/src/main/scala/org/acme/{resource.class-name}.tpl.qute.scala
package org.acme

import javax.ws.rs.\{GET, Path, Produces}
import javax.ws.rs.core.MediaType

@Path("{resource.path}")
class {resource.class-name} {

    @GET
    @Produces(Array[String](MediaType.TEXT_PLAIN))
    def hello() = "{resource.response}"
}

File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/resteasy-codestart/java/src/main/java/org/acme/{resource.class-name}.tpl.qute.java
package org.acme;

{#if quarkus.bom.version.startsWith("2.") or quarkus.bom.version.startsWith("1.")}
import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;
{#else}
import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.Produces;
import jakarta.ws.rs.core.MediaType;
{/if}

@Path("{resource.path}")
public class {resource.class-name} {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String hello() {
        return "{resource.response}";
    }
}
